sky/server/requests/executor.py

```diff
@@ -73,8 +73,13 @@
 # Constants based on profiling the peak memory usage of
 # various sky commands. See `tests/load_test/` for details.
 # Max memory consumption for each request.
-_PER_BLOCKING_REQUEST_MEM_GB = 0.25
-_PER_NON_BLOCKING_REQUEST_MEM_GB = 0.15
+# NOTE(dev): update these constants for each release according to the load
+# test results.
+# TODO(aylei): maintaining these constants is error-prone, we may need to
+# automatically tune parallelism at runtime according to system usage stats
+# in the future.
+_PER_BLOCKING_REQUEST_MEM_GB = 0.4
+_PER_NON_BLOCKING_REQUEST_MEM_GB = 0.25
```

Comment:
we could also add a note that these values can vary significantly depending on which clouds are enabled, which I observed during load testing on jobs controller
FWIW the "equivalent" value for jobs controller for `_PER_BLOCKING_REQUEST_MEM_GB` is set to 350MiB at the moment, which is fine for AWS but a bit sketchy on some other clouds
https://github.com/skypilot-org/skypilot/blob/a759f24649aa9faff5b88f21b8f08b367c55c170/sky/jobs/scheduler.py#L63-L64